src/store.js

```
@@ -0,0 +1,56 @@
+import { hashHistory } from 'react-router';
+import { routerReducer, routerMiddleware } from 'react-router-redux';
+import { combineReducers, createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+
+import settings from './settings';
+
+const enhancers = [];
+const middleware = [thunk];
+if (window) {
+  if (window.devToolsExtension) {
+    enhancers.push(window.devToolsExtension());
+  }
+}
+
+const configuredRouterMiddleware = routerMiddleware(hashHistory);
+middleware.push(configuredRouterMiddleware);
+
+// TODO: make logger optional
+let createLogger;
+let logger;
+try {
+  createLogger = require('redux-logger').default;
+
+  logger = createLogger({
+    duration: true,
+  });
+  middleware.push(logger);
+}
+catch (e) {
+  // do nothing
```

I have removed that part, I don't like it so no more empty catch